src/main/resources/assets/css/style.css

```diff
@@ -13,6 +13,12 @@
 
 /* --- */
 
+.swal2-container {
+  font-family: "Roboto";
+}
+
+/* --- */
```

Comment:
/* --- */ 

Seperator code? would not a comment/explination above add more info and seperate it?

Reply:
I agree with you! I'll add small comments to explain why I'm separating into those sections!